### PR TITLE
feat(gallery): add gallery page

### LINKED ISSUE #116: Page image galerie
Ajouter un nouveau bouton sur le home page « Files Gallery » qui redirige vers « /file-gallery ».

Un champ Input text : Chemin du répertoire de la galerie. 
exemples : 
- "../licence/images"
- "../ri/images"

Avec un bouton « validate » 

Si le chemin n’existe pas => remonter un message 
Directory does not exist.
Si pas de fichier => remonter le message :
No file in this directory.

S’il existe afficher la galerie d’image (.gif, .tiff, jpg, jpeg, png) avec un aperçu pour les images..
Pour les autres type de fichiers, on affiche une icone générique.

Trie => « du plus récent au plus vieux », « du plus vieux au plus récent », par ordre alphabetique desc, par ordre alphabetique asc.
Taille => max width + max height: 64 px, 128px, 256px, 512px

Au clic sur une image, elle s’affiche dans le navigateur en taille normal (sortie complète de l’interface ml-cli). L’utilisateur devra faire clic droit open in new page pour en ouvrir plein.
Au clic sur un pdf, il s'affiche dans le navigateur.
Au clic sur les autres type de fichier, ils sont téléchargés.

La page doit se rafraichir toutes les 5 secondes (c'est à dire si des nouvelles images sont apparait dans le répertoire, elles doivent apparaître).

image

### PR DESCRIPTION
**Pull request by lilian-delouvy**
closes #116

This pull request adds the galery page to Ml-Cli:
l3MMfAG2rU

### CODE REVIEW

File: demo/tasks-licenses.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
 	},
 	{
 		"type": "dataset",
-		"enabled": true,
+		"enabled": false,
```
Comment (guillaume-chervet) [inline demo/tasks-licenses.json:115]: Is it what you Want?

**Review by guillaume-chervet — CHANGES_REQUESTED** [review]

File: src/Ml.Cli.WebApp/ClientApp/src/Local/Comparison/__snapshots__/index.spec.js.snap [hunk 2]
```diff
@@ -1239,7 +1239,7 @@ exports[`Check a compare file insertion Find table result of KOs after file inpu
     </div>
   </div>
   <div
-    class="af-title-bar af-title-bar--backhome"
+    class="af-title-bar af-title-bar--backhome "
```
Comment (guillaume-chervet) [inline src/Ml.Cli.WebApp/ClientApp/src/Local/Comparison/__snapshots__/index.spec.js.snap:1242]: Idem?

File: src/Ml.Cli.WebApp/ClientApp/src/Local/Gallery/Gallery.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+const loadStateAsync = (fetch) => async (setState, state) => {
+    setState({
+        ...state,
+        status: resilienceStatus.LOADING,
+    });
+    const uri = encodeURI("/api/gallery/" + state.filesPath);
+    const response = await fetchGetData(fetch)(uri);
+    if (response.status >= 300) {
```
Comment (guillaume-chervet) [inline src/Ml.Cli.WebApp/ClientApp/src/Local/Gallery/Gallery.js:40]: Why 300? May be 400 would be better

**Review by guillaume-chervet — COMMENTED** [review]
C'est top

File: src/Ml.Cli.WebApp/ClientApp/src/Local/Gallery/Gallery.js (hunk 3 above)
Comment (guillaume-chervet) [reply src/Ml.Cli.WebApp/ClientApp/src/Local/Gallery/Gallery.js:40]: Or the inverse between 200 and <300

File: demo/tasks-licenses.json (hunk 1 above)
Comment (lilian-delouvy) [reply demo/tasks-licenses.json:115]: That was supposed to be a temporary measure to work without launching the background tasks. I made a new commit to enable the tasks

File: src/Ml.Cli.WebApp/ClientApp/src/Local/Comparison/__snapshots__/index.spec.js.snap (hunk 2 above)
Comment (lilian-delouvy) [reply src/Ml.Cli.WebApp/ClientApp/src/Local/Comparison/__snapshots__/index.spec.js.snap]: That was due to the acceptance of classmodifiers for the titlebar. I updated it !

File: src/Ml.Cli.WebApp/ClientApp/src/Local/Gallery/Gallery.js (hunk 3 above)
Comment (lilian-delouvy) [reply src/Ml.Cli.WebApp/ClientApp/src/Local/Gallery/Gallery.js]: Updated to 400

**Review by guillaume-chervet — APPROVED** [review]

**Pull request merged**